ibft/pipeline/auth/msg_lambda.go

```diff
@@ -18,3 +19,17 @@ func ValidateLambdas(lambda []byte) pipeline.Pipeline {
 		return nil
 	})
 }
+
+// ValidateLambdasRole validates msg role with ibft controller role
+func ValidateLambdasRole(role string) pipeline.Pipeline {
```

Comment:
@nivBlox iBFT doesn't know what role is.. let's keep it iBFT generic.. this should be ValidateLambdas